### PR TITLE
Handover fix for NH, NCC

### PR DESCRIPTION
**Pull request by badhrinathpa**

### CODE REVIEW

File: include/common/s1ap_structs.h [hunk 1]
```diff
@@ -172,7 +172,7 @@ typedef struct targetId{
         struct TAI selected_tai;
 }targetId;
 
-#define TRANS_CONT_SIZE 2700
+#define TRANS_CONT_SIZE 3000
```
Comment (thakurajayL) [inline include/common/s1ap_structs.h:175]: should we just make it 4k ?

File: src/mme-app/actionHandlers/attachActionHandlers.cpp [hunk 2]
```diff
@@ -926,11 +926,13 @@ ActStatus ActionHandlers::send_init_ctxt_req_to_ue(SM::ControlBlock& cb)
...
 	secinfo& secInfo = const_cast<secinfo&>(ue_ctxt->getUeSecInfo().secinfo_m);
 
-	SecUtils::create_kenb_key(secVect->kasme.val, secInfo.kenb_key, nas_count);
-	secInfo.next_hop_chaining_count = 0 ;
-	memcpy(secInfo.next_hop_nh , secInfo.kenb_key, KENB_SIZE);
-	
-	init_ctx_req_Q_msg icr_msg;
+    SecUtils::create_kenb_key(secVect->kasme.val, secInfo.kenb_key, nas_count);
+    unsigned char nh[SECURITY_KEY_SIZE] = { 0 };
+    secInfo.next_hop_chaining_count = 1;
```
Comment (thakurajayL) [inline src/mme-app/actionHandlers/attachActionHandlers.cpp:931]: what does it mean ?

File: src/mme-app/sec/secUtils.cpp [hunk 3]
```diff
@@ -114,7 +114,7 @@ void SecUtils::create_nh_key(const unsigned char *kasme, unsigned char *nh_key,
 {
        uint8_t salt[35] = {0};
        salt[0] = NEXT_HOP_FC; /*TODO : Sec algo. handle properly instead of harcoding here*/
```
Comment (thakurajayL) [inline src/mme-app/sec/secUtils.cpp:116]: Impact of this comment ?

Comment (badhrinathpa) [reply src/mme-app/sec/secUtils.cpp:116]: this value is constant for 4g. Don't think it has an impact.

File: src/mme-app/actionHandlers/attachActionHandlers.cpp (hunk 2 above)
Comment (badhrinathpa) [reply src/mme-app/actionHandlers/attachActionHandlers.cpp:931]: Previously we used to store the KenB as the NH for NCC value 0. Instead we should store the NH value for NCC value 1.

File: include/common/s1ap_structs.h (hunk 1 above)
Comment (badhrinathpa) [reply include/common/s1ap_structs.h:175]: we can update if needed. currently it is able to handle the src to target container size for iPhone and Samsung.

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**